backend/routes/api.php

```diff
@@ -20,3 +21,5 @@
 });
 
 Route::resource('users', UserController::class, ['only' => ['index']])->middleware(['admin', 'auth']);
+
+Route::resource('categories', CategoryController::class)->middleware(['admin', 'auth']);
```

Comment:
can you try to group middlewares? since both are using the same

https://laravel.com/docs/master/middleware#middleware-groups